cnf-certification-test/identifiers/identifiers.go

```diff
@@ -36,6 +36,10 @@ const (
 const (
 	TagCommon   = "common"
 	TagExtended = "extended"
+	TagTelco    = "telco"
```

Comment:
Should we then remove the "common" and "extended" tags? I think there is some overlap with "general" and "vz".

Reply:
I agree, I figured we would discuss that as well as part of this PR.